services/QuillLMS/bin/spring

```diff
@@ -5,12 +5,14 @@
 # It gets overwritten when you run the `spring binstub` command.
 
 unless defined?(Spring)
-  require "rubygems"
-  require "bundler"
+  require 'rubygems'
```

Comment:
Have you confirmed that this code will not run spring in the prod environment? 

There are special instructions for avoiding loading spring in prod, here: https://github.com/rails/spring#deployment

Reply:
Note, this code isn't adding `spring`. It's already in the `Gemfile`. This code just edits the config for caching classes in `test`, and updates CircleCI to use `bin/rspec`.

Heroku already skips loading `development` and `test` gems (but again, nothing is changing here):


![Screen Shot 2022-09-26 at 1 59 00 PM](https://user-images.githubusercontent.com/1304933/192347812-0d334c97-6352-4c57-a508-f87ebf668880.png)